policies/terms_of_membership.md

```diff
@@ -0,0 +1,113 @@
+
+Unless otherwise agreed by members, no members will receive a fee or discount to dues/desk rends for carrying out tasks related to the running of the building or charity.
+
+## General Rules
+
+- Farset Labs is a shared resource. Members know that the building may be occupied by others at any time, including, but not limited to activities of Farset Labs.
+- Farset Labs is not liable for any injury or lost/stolen property from the space.
+- Members are fully responsible for the actions of their guests whilst in the building
+- Members are responsible for replacing any broken or damaged property caused by member or member’s guests.
+- All areas of the building are subject to inspection by the building owners at any time and must provide 24hrs notice before doing so unless deemed an emergency. 
+
+
+
+## Shared Spaces & Facilities for hire
+
+- Please contact events@farsetlabs.org.uk to book rooms.
+- Room availability can be checked online at https://www.farsetlabs.org.uk/events/room_availability.html
+
+Certain shared facilities are available to hire by non-members. These areas include the downstairs event space,the pod space and the McKee Meeting Room. Members have free access to use these spaces during times when they are not booked by paying non-members. 
+
+All shared spaces must be left tidy and clean at the end of each day. Work, tools, equipment and materials should be stored in member’s own storage boxes and not left in shared spaces.
+
+### Insurance
+
+- Farset Labs will provides public liability insurance for common areas only for activities operated by agents of the charity, i.e. members running Farset Labs event. Independent/Outside booking should establish their own PLI requirements.
+- Insurance for member-owned equipment are not covered by Farset Labs and members leave items in the workshop and common areas at their own risk.
+
+### Health & Safety
+
+- For accidents, health crises, criminal activities or fires call 999.
+- For any urgent issue that might compromise the structure of safety of the building, please contact the Weavers Court security desk, directly across from the front doors.
+
+### Fire Safety
+
+- Members agree to adhere to all fire regulations as detailed in the Fire Safety Policy.
+- Strictly no smoking or open flames anywhere in the building.
+- Members and their guests must sign in/out of the building using the system provided. #todo this would be a bit change but given the shift in size of building might not be a bad idea...
+
+### Security
+
+- Members agree to adhere to all security protocols as detailed in the Induction 
+- Members assume responsibility for the security of the building, and their property.
+- No propping of the main door or other external doors, which must remain closed at all times.
```

Comment:
"Members must not prop open or otherwise attempt to leave open the main door..."